pkg/registryclient/client.go

```diff
@@ -137,19 +137,19 @@ func WithCredentialHelpers(credentialHelpers ...string) Option {
 	return func(c *config) error {
 		var chains []authn.Keychain
 		helpers := sets.New(credentialHelpers...)
-		if helpers.Has("default") {
+		if helpers.Has("DEFAULT") {
```

Comment:
Helpers are also passed by flags.

Reply:
i dont understand this can you please explain it.